src/main/java/org/jenkinsci/plugins/workflow/steps/StepContext.java

```diff
@@ -149,4 +149,9 @@ public boolean hasBody() {
     public abstract int hashCode();
 
     private static final long serialVersionUID = 1L;
+
+    /**
+     * Returns the function name used in pipeline Jenkinsfile for the current StepNode.
```

Comment:
```suggestion
     * Returns the function name used in pipeline Jenkinsfile for the current {@link StepNode}.
```

Reply:
now removed the method in favor of using `DefaultStepContext#get(<key>)` method way